documentation/doc-Planning_your_migration/assemblies/assembly_live-migration.adoc

```diff
@@ -6,11 +6,7 @@
 //:context: cold-warm-migration
 
 [role="_abstract"]
-You can use _live migration_ to migrate virtual machines (VMs) between {virt} clusters, or between namespaces on the same {virt} cluster, with extremely limited downtime.
-
-Live migration is supported by {project-first} version 2.10.0 and later. It requires {virt} 4.20 or later on both your source and target clusters. 
-
-Live migration makes it easier for you to perform Day 2 tasks, such as seamless maintenance and workload balancing after you have migrated your VMs to {virt}.
+To reduce downtime during Day 2 operations like maintenance and workload balancing, migrate VMs between {virt} clusters or namespaces by using live migration. You need {project-first} 2.10.0 or later and {virt} 4.20 or later on both clusters.
 
```

Comment:
Please leave the original  two sentences as is.  Live migration is always about  reducing downtime for CNV-CNV migration, whether the user is thinking in terms of "Day 2 operations" or not.  "Both" is problematic because live migration is also a solution for migration between namespaces on the same cluster (source and target cluster are the same).